filebeat/prospector/prospector_stdin.go

```diff
@@ -8,16 +8,15 @@ import (
 )
 
 type ProspectorStdin struct {
-	Prospector *Prospector
-	harvester  *harvester.Harvester
-	started    bool
+	harvester *harvester.Harvester
+	started   bool
```

Comment:
I remember I introduced it because there was an issue in the past that it was started twice. Will have to check in more detail if it is still needed. Lots of changes have happened ;-) Will put in my list for future clean ups.